lua/tests/helpers.lua

```diff
@@ -152,15 +152,18 @@ end
 assert:register('assertion', 'contents', has_buf_contents)
 
 --- Assert about the current infoview contents.
-local function has_infoview_contents(_, arguments)
-  local expected = dedent(arguments[1][1] or arguments[1])
-  local target_infoview = arguments[1].infoview or infoview.get_current_infoview()
-  local got = table.concat(target_infoview:get_lines(), '\n')
-  assert.are.same(expected, got)
-  return true
+local function has_infoview_contents(diff)
```

Comment:
(Same here on boolean arguments, I'd prefer this was 2 separate functions which possibly call a helper function that does most of the work)